tests/icons.test.mjs

```diff
@@ -1,14 +1,30 @@
+import fs from 'fs';
+import path from 'path';
 import simpleIconsData from '../_data/simple-icons.json';
 import utils from '../scripts/utils.js';
 import * as simpleIcons from '../icons.mjs';
 import testIcon from './test-icon.js';
 
 const { getIconSlug, slugToVariableName } = utils;
 
+const iconsDir = path.resolve(process.cwd(), 'icons');
+
 simpleIconsData.icons.forEach((icon) => {
   const slug = getIconSlug(icon);
   const variableName = slugToVariableName(slug);
   const subject = simpleIcons[variableName];
 
   testIcon(icon, subject, slug);
+
+  const svgPath = path.resolve(iconsDir, `${slug}.svg`);
+
+  test(`${icon.title} has a valid svg value`, () => {
+    const svgFileContents = fs
+      .readFileSync(svgPath, 'utf8')
+      .replace(/\r?\n/, '');
+
+    expect(subject.svg.substring(subject.svg.indexOf('<title>'))).toEqual(
+      svgFileContents.substring(svgFileContents.indexOf('<title>')),
+    );
+  });
```

Comment:
What is the purpose of these lines? Is already tested [here](https://github.com/simple-icons/simple-icons/blob/7c8146e475ffaf3a0c77d1c5c88a42686ef2399b/tests/test-icon.js#L61-L71), right?

Reply:
I guess my conflict fixing skills were not as good as I thought they were :laughing: